### PR TITLE
Comment the Lighthouse JSON manifest as HTML in the related GH PR

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at commenting the Lighthouse JSON manifest as HTML in the related GH PR.

The bot will create a new message each time, to be able to compare the summaries in between commits.

image

image

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-ap1ah5 [lighthouse] - Comment manifest to GH relative PR in HTML](https://app.clickup.com/t/ap1ah5)

**Comment by Fuzznimp** [comment]
<details>
        <summary>Click here to see the Lighthouse manifest</summary>
    <p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/profile/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/profile/user-profile/new/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/logins/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/logins/email/new/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/logins/phone/new/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/security/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/security/sudo/</p><p>https://fewlines-acc-cu-ap1ah5--fkjkdd.herokuapp.com/account/locale/</p></details>

**Comment by Fuzznimp** [comment]
<details>
        <summary>Click here to see the Lighthouse manifest</summary>
    <p>:</p></br>0.88</br>1</br>1</br>1</br><p>account/:</p></br>0.96</br>0.9</br>1</br>1</br><p>account/profile/:</p></br>0.91</br>0.87</br>1</br>1</br><p>account/profile/user-profile/new/:</p></br>0.88</br>0.9</br>1</br>1</br><p>account/logins/:</p></br>0.93</br>0.89</br>1</br>1</br><p>account/logins/email/new/:</p></br>0.83</br>0.9</br>1</br>1</br><p>account/logins/phone/new/:</p></br>0.89</br>0.91</br>1</br>1</br><p>account/security/:</p></br>0.9</br>0.88</br>1</br>1</br><p>account/security/sudo/:</p></br>0.92</br>0.9</br>1</br>1</br><p>account/locale/:</p></br>0.92</br>0.92</br>1</br>1</br></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><p>/:</p><br>0.91<br>1<br>1<br>1<br></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><p>/:</p><ul><li>Performance: 0.91</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1<li></ul></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><p><strong>/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.87</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.89</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><p><strong>/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.87</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.89</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.87</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.89</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.87</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by Oluane** [comment]
Neat 🎉

File: bin/lighthouse/lh-manifest-to-html.mjs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  return fs.writeFile(
+    path.join(process.cwd(), "./html-manifest.txt"),
+    HTMLManifest,
+    (error) => {
+      if (error) {
+        throw error;
+      }
+
+      console.log("html-manifest.txt saved.");
```
Comment (Oluane) [inline bin/lighthouse/lh-manifest-to-html.mjs:37]: Is it a dev leftover or is it intended ?

Comment (Fuzznimp) [reply bin/lighthouse/lh-manifest-to-html.mjs:37]: It is intended, but I can remove it if you prefer. Not sure it adds real value 🤔.

Comment (Oluane) [reply bin/lighthouse/lh-manifest-to-html.mjs:37]: I suppose that if something goes wrong there, the error will be thrown, so idk if the console log adds something indeed 🤷‍♀️

Comment (Fuzznimp) [reply bin/lighthouse/lh-manifest-to-html.mjs:37]: Fair point 😄.

Comment (Fuzznimp) [reply bin/lighthouse/lh-manifest-to-html.mjs]: Fixed 👍

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.87</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.89</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Review by Oluane — APPROVED** [review]

**Pull request merged**